drivers/hmis/app/graphql/types/hmis_schema/client_alert.rb

```diff
@@ -0,0 +1,19 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::ClientAlert < Types::BaseObject
+    description 'Alert'
+    field :id, ID, null: false
+    field :note, String, null: false
+    field :created_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :created_by, Types::Application::User, null: true
+    field :expiration_date, GraphQL::Types::ISO8601DateTime, null: true
```

Comment:
I think the expiration date was stored as a date not a datetime, right? If so it should resolve a `ISO8601Date`

Reply:
That's right! Thank you :)